css/style.css

```diff
@@ -0,0 +1,149 @@
+body {
+    margin: 0;
+}
+
+/*float right and left*/
+.right {
+    float: right;
+}
+
+.left {
+    float: left;
+}
+
+.wrap {
+    margin: 0 auto;
+    max-width: 1150px;
+    padding: 20px;
+}
+
+/*header*/
+header {
+    text-align: center;
+    border-bottom: 1px solid #ddd;
+    padding: 10px 0;
+}
+
+header a {
+    text-decoration: none;
+    color: #000000;
+}
+
+.nav {
+    margin: 0 10px;
+}
+
+.logo {
+    width: auto;
+    height: 28px;
+    vertical-align: bottom;
+}
+
+header button {
+    color: white;
+    background-color: #6288a5;
+    border: 1px solid #326891;
+    font-weight: bold;
+    padding: 3px 9px;
+}
+
+/*************  Main div **************/
+
+.article-img img{
+    max-width: 100%;
+}
+
+article h1 {
+    margin: 20px auto;
+    max-width: 600px;
+    font-family: 'Times New Roman', Times, serif;
+    font-style: italic;
+}
+
+.author {
+    font-weight: 700;
+    font-family: Helvetica, Arial, sans-serif;
+    font-size: 0.9375rem;
+    line-height: 1.25rem;
+}
+
+.date {
+    margin: 0;
+    float: left;
+    color: #333;
+    font-weight: 500;
+}
+
+.icons {
+    float: right;
+}
+
+.icons i {
+    margin-left: 10px;
+}
+
+
+article p {
+    clear: both;
+}
+
+.text {
+    margin-left: calc((100% - 600px) / 2);
```

Comment:
margin is still to0 big.
remember text-align center :)

Reply:
text-align doesn't work in this case, because we want to center the paragraph not the actual text

Reply:
ok we fixed it with:
`margin: 0 auto;`